Assets/Scripts/CardController.cs

```diff
@@ -28,19 +29,16 @@ public class CardController : MonoBehaviour {
         handController.HandleCardClick (this, isSelected);
     }
 
-    public void SetSelected(bool selected) {
+    public void SetSelected (bool selected) {
         this.isSelected = selected;
     }
 
-    public void UpdateCardDisplay (int newIndex, bool canCast = false) {
+    public void UpdateCardDisplay () {
```

Comment:
Simplified to just changing the selection display.  Things were getting broken when it did too many other imperative calls.